src/model.js

```diff
@@ -0,0 +1,6 @@
+import falcor from '@graphistry/falcor';
```

Comment:
It looks like that if I have:

`import falcor from 'falcor';`

everything works, but if I have:

`import falcor from '@graphistry/falcor';`

I've got this error:

```console
 error RangeError: Maximum call stack size exceeded
    at Array.forEach (native)
    at n (http://127.0.0.1:3000/bundle.js:24:7125)
    at r (http://127.0.0.1:3000/bundle.js:24:7041)
    at Object.get (http://127.0.0.1:3000/bundle.js:24:7226)
    at derez (<anonymous>:2:5752)
    at derez (<anonymous>:2:5745)
    at derez (<anonymous>:2:5745)
    at derez (<anonymous>:2:5745)
    at derez (<anonymous>:2:5745)
    at derez (<anonymous>:2:5745)
```

I'm pretty sure it is something related w/ the `DataSource` ...